Assets/Scripts/MenuBehavior.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuBehavior : MonoBehaviour
+{
+    // THIS IS A PLACEHOLDER BEFORE WE HAVE A LEVEL SELECT
+    // TODO: REPLACE THIS WITH A PROPER LEVEL SELECT
+    
+    public string firstLevel;
+    public Animator fadeToBlackAnimation;
+    public AudioClip buttonClickSfx;
+
+    private static readonly int FadeOut = Animator.StringToHash("FadeOut");
+
+    private Transform mainCam;
+    private bool menuOptionSelected = false;
+
+    private void Awake()
+    {
+        mainCam = Camera.main.transform;
```

Comment:
You can just reference `LevelManager.Camera` here. Referencing `Camera.main` more times than you have to can be slow.